src/test/java/co/edu/icesi/seleniumworkshop/steps/FacebookStepDef.java

```diff
@@ -1,40 +1,55 @@
 package co.edu.icesi.seleniumworkshop.steps;
 
 import co.edu.icesi.seleniumworkshop.common.TestContext;
+import co.edu.icesi.seleniumworkshop.pages.SearchPage;
+import co.edu.icesi.seleniumworkshop.pages.LoginPage;
 import cucumber.api.java.After;
 import cucumber.api.java.Before;
 import cucumber.api.java.en.Given;
 import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
 import org.openqa.selenium.WebDriver;
 
 import java.util.concurrent.TimeUnit;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 public class FacebookStepDef {
 
     private TestContext testsContext = new TestContext();
-    private WebDriver webDriver;
+    private WebDriver webDriver = testsContext.getDriver();
+
+    private LoginPage loginPage = new LoginPage(webDriver);
+    private SearchPage searchPage = new SearchPage(webDriver);
 
     @Before
     public void setUp() {
-        webDriver = testsContext.getDriver();
-        webDriver.manage().timeouts().implicitlyWait(2, TimeUnit.SECONDS);
+        webDriver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
     }
 
     @After
     public void tearDown() {
+        webDriver.close();
         webDriver.quit();
         webDriver = null;
     }
 
-    @Given("^I am in the Facebook's Home Page$")
+    @Given("^a user logged in the application$")
     public void givenLoggedInHomePage() {
         webDriver.get(testsContext.getBaseUrl());
+        String email = testsContext.getEmail();
+        String password = testsContext.getPassword();
+        loginPage.login("leonleo997@gmail.com", "yleonardo97");
```

Comment:
use email and password variables instead of your credentials

Reply:
HAHAHAHAHAHA